Bugzilla/Template.pm

```diff
@@ -975,6 +868,14 @@ sub create {
             # Function for retrieving global parameters.
             'Param' => sub { return Bugzilla->params->{$_[0]}; },
 
+            asset_file => sub {
+                return Bugzilla->asset_manager->asset_file($_[0]);
+            },
+
+            asset_files => sub {
+                return [Bugzilla->asset_manager->asset_files(@{ $_[0] })];
```

Comment:
It weirds me out that the [] is placed here instead of upstream, but it clearly works, so shrug.